lib/compress/zstd_compress.c

```diff
@@ -2806,14 +2806,13 @@ ZSTD_CStream* ZSTD_createCStream_advanced(ZSTD_customMem customMem)
     return zcs;
 }
 
-size_t ZSTD_freeCStream(ZSTD_CStream* zcs)
+void ZSTD_freeCStream(ZSTD_CStream* zcs)
 {
-    if (zcs==NULL) return 0;   /* support free on NULL */
+    if (zcs==NULL) return;   /* support free on NULL */
```

Comment:
If the return type is void, what purpose does "return" still have? Does it kind of work like `break`?